src/app/AlwaysOn.HealthService/README.md

```diff
@@ -1,89 +1,6 @@
 # Health Service
 
-The HealthService is an app component that is running along other components like CatalogService and BackgroundProcessor on the compute cluster. It provides a REST API that is being called by Azure Front Door to determine the health of a stamp (region). Unlike basic liveness probes, which are present on every API, health service is a more complex component which reflects the state of dependencies, in addition to its own.
-
-![HealthService conceptual diagram](/docs/media/health-service-high-level.png)
-
-The idea is, first of all, if the cluster itself is down, the health service won't respond at all. When the service is up and running, it performs periodic checks against various components of the solution:
-
-- It attempts to do a simple query against Cosmos DB
-- It attempts to send a message to Event Hub (the message will be filtered out by the background worker)
-- It looks up a state file on the storage account. This file can be used to turn off a region, even while the other checks are still working ok.
-
-All health check results are cached in memory for a configurable number of seconds (by default 10) so that not every call to the API results in backend calls. While this does add a small potential latency in detecting outages, it also reduces the additional cluster load generated by health checks.
-
-## Configuration
```

Comment:
this section (and the following ones) are super specific to the implementation which is here on github. I would prefer to keep it here (and probably remove it from docs)

Reply:
Yes, that makes sense - but its already on docs.microsoft.com (see [here](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks-mission-critical/mission-critical-health-modeling)) - we've discussed with @PageWriter-MSFT that it might make sense to cleanup the repo here now and port/move content back from docs into the GH repos as a next step.

Otherwise we've to align changes on docs and here.

Reply:
understood. But to remove stuff now that we know we want to put back later... I'm not onboard with that ;) I'd rather have content duplicate for the time being in docs...

Reply:
No you got me wrong - we haven't decided yet what's too detailed for docs.microsoft.com - and haven't started to migrate anything back - though this part should definitely taken into consideration.

For now this PR is suppposed to cleanup content that's now available on docs.microsoft.com. I don't want to overcomplicate things here.

Reply:
Is this specific file the only one that is blocking us from getting this PR merged? I'd then probably just leave it as it is/was for now.

Reply:
yes, if we can leave this file untouched for now, I'm happy :)